subprojects/gdk_rust/gdk_electrum/src/store.rs

```diff
@@ -477,11 +413,16 @@ mod tests {
         )
         .unwrap();
 
-        let mut store = StoreMeta::new(&dir, xpub, None, id).unwrap();
-        store.cache.heights.insert(txid, Some(1));
-        drop(store);
+        let id = NetworkId::Bitcoin(Network::Testnet);
+
+        {
+            let mut store = StoreMeta::new(&dir, xpub, id).unwrap();
```

Comment:
`test_db_roundtrip` is currently failing, needs a call to `make_account_cache` I guess

Reply:
Addressed by https://github.com/Blockstream/gdk/pull/93/commits/80414ebd6efed1ba475490ce610ac6c387a650f8

Reply:
Actually this broke `./launch_integration_tests.sh bitcoin`

Reply:
https://github.com/Blockstream/gdk/commit/80414ebd6efed1ba475490ce610ac6c387a650f8 uncovered a deadlock during account initialization, now fixed by https://github.com/Blockstream/gdk/pull/93/commits/886083bee1c461ad1cb6abb5db42bc4256c4e164.